src/components/collapsible_nav/collapsible_nav.tsx

```diff
@@ -95,7 +95,8 @@ export const EuiCollapsibleNav: FunctionComponent<EuiCollapsibleNavProps> = ({
 }) => {
   const [flyoutID] = useState(id || htmlIdGenerator()('euiCollapsibleNav'));
   const [windowIsLargeEnoughToDock, setWindowIsLargeEnoughToDock] = useState(
-    window.innerWidth >= dockedBreakpoint
+    (typeof window === 'undefined' ? -Infinity : window.innerWidth) >=
```

Comment:
For this one, I'd actually like to do the opposite of breadcrumbs and ensure that it will always allow docking.
```suggestion
    (typeof window === 'undefined' ? Infinity : window.innerWidth) >=
```